eng/pipelines/client.yml

```diff
@@ -89,30 +89,17 @@ jobs:
           goals: 'install'
 
       - task: Maven@3
-        displayName: 'Install all client libraries (for SpotBugs analysis)'
+        displayName: 'Install Module and Run SpotBugs, Checkstyle, and Javadoc'
         inputs:
-          mavenPomFile: pom.client.xml
-          options: '$(DefaultOptions) -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true'
+          mavenPomFile: $(pomFile)
+          options: '$(DefaultOptions) -DskipTests -Dgpg.skip -Dverify-readme
```

Comment:
I am curious if `-Dverify-readme` worked because core's `pom.service.xml` just uses `pom.client.xml`. It uses relative path and it would be good to test this out with a different `pom.service.xml` like appconfig and see the logs to confirm verify-readme is looking at the right files.

Reply:
Great question, I will run these changes with adding a Checkstyle, SpotBugs, Javadoc, and README verification error to validate that it will catch all of them.

Reply:
Updated to work with running `pom.client.xml` and `pom.service.xml`.